packages/cli/src/options/beaconNodeOptions/chain.ts

```diff
@@ -123,11 +125,19 @@ Will double processing times. Use only for debugging purposes.",
     group: "chain",
   },
 
-  "chain.proposerBoostEnabled": {
+  "chain.proposerBoost": {
```

Comment:
as this is an existing flag, we must add the prev. value as an alias (even though I doubt anyone is using it)

At some point, when we do a v2 release, we can clean those up